src/api/uploads/FolderUploadNode.js

```diff
@@ -231,6 +263,14 @@ class FolderUploadNode {
             this.readEntry(reader, resolve);
         });
     };
+
+    /**
+     * Returns the folderId
+     * @returns {string}
+     */
+    getFolderId = (): string => {
```

Comment:
`?string` ?

Reply:
It returns an empty string if not set, does that still need a maybe type?